src/client/cypress/e2e/helpers/formHelpers.js

```diff
@@ -197,12 +188,7 @@ export const evaluateYesNoSelect = (fieldName, expectedValue, parent) => {
 export const evaluateSelect = (fieldName, expectedText, parent = null, editable = true) => {
   if (editable) {
     const selector = createBaseSelector(parent) + `[data-cy="${fieldName}-formSelect"] input`;
-    cy.get(selector).then($input => {
```

Comment:
FYI @danjov @tschumpr @domi-b Wir hatten ein Problem mit unseren Evaluate Funktionen was einen Teil der Flakyness unserer Tests erklärt.
Cypress hat ja einen automatischen retry und timeout Mechanismus für seine commands. In den Evaluate Funktionen in diesem File haben wir aber nur mit cy.get(selector) cypress commands genutzt. Das eigentliche Holen und Evaluieren des Inputwerts waren keine cypress Funktionen (sondern im Beispiel hier jquery) und deshalb auch vom Retry ausgenommen. 
Der retry hat sich also nur auf das Auffinden des Inputs aber nicht auf das Evaluieren des Werts bezogen.